metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -186,10 +187,30 @@ def _set_obj_xyz(self, pos):
         qvel[9:15] = 0
         self.set_state(qpos, qvel)
 
-    def get_site_pos(self, siteName):
+    def _get_site_pos(self, siteName):
         _id = self.model.site_names.index(siteName)
         return self.data.site_xpos[_id].copy()
 
+    def _set_pos_site(self, name, pos):
+        """Sets the position of the site corresponding to `name`
+
+        Args:
+            name (str): The site's name
+            pos (np.ndarray): Flat, 3 element array indicating site's location
+        """
+        assert isinstance(pos, np.ndarray)
+        assert pos.ndim == 1
+
+        self.data.site_xpos[self.model.site_name2id(name)] = pos[:3]
+
+    @property
+    def _target_site_config(self):
+        """Retrieves site name(s) and position(s) corresponding to env targets
+
+        :rtype: list of (str, np.ndarray)
+        """
+        return [('goal', self._state_goal)]
```

Comment:
shouldn't this also return the object initial position? My understanding of this function is that it would be used to return the identifying information of a task configuration for a task, which would be not only the goal, but also the object initial position.

Reply:
No it's just to specify where the markers should go. Most environments only have 1 goal marker, and it gets put at `_state_goal`, which is why I have this default implementation. A few have 2 goal markers, so they override this and return `('name_of_marker', pos_of_marker)`